packages/plugin-core/src/test/suite-integ/ConvertLink.test.ts

```diff
@@ -18,6 +19,23 @@ import { WSUtils } from "../../WSUtils";
 import { expect } from "../testUtilsv2";
 import { describeMultiWS, setupBeforeAfter } from "../testUtilsV3";
 
+const getReference = async ({
+  editor,
+  position,
+}: {
+  editor: TextEditor;
+  position: vscode.Position;
+}) => {
+  const { wsRoot, vaults } = ExtensionProvider.getDWorkspace();
+  const out = await getReferenceAtPosition({
+    document: editor.document,
+    position,
+    wsRoot,
+    vaults,
+  });
+  return out as getReferenceAtPositionResp;
```

Comment:
It might be good to add a `expect(out).toBeTruthy()` here so the test will fail early if there's an issue.

Reply:
added a check